src/golang/lib/engine/authenticate.go

```diff
@@ -34,37 +33,6 @@ func AuthenticateK8sConfig(ctx context.Context, authConf auth.Config) error {
 	return nil
 }
 
-func AuthenticateLambdaConfig(ctx context.Context, authConf auth.Config) error {
```

Comment:
The `AuthenticateLambdaConfig` now becomes `ConnectToLambda`. The Authenticate Function is basically taking the RoleArn from users input and create lambda functions, which is technically authentication and connecting to integrations combined. Now that we want to connect to lambda async, we want to have the integration immediately created when connecting to Lambda, so the authentication of the config is passed

Reply:
This makes sense to me. @hsubbaraj-spiral could you also check if this looks good to you?

Reply:
Yes, this is ok with me! Since we parse the Config in ConnectToLambda, it is redundant to just have the check that the provided struct is correct. @Fanjia-Yan can you look into how we can verify that the role exists (I think a check that the role exists for now is ok, there's no way of making sure that the role has the correct permissions until we actually create the lambda fxn) as a part of validateLambdaConfig? That can be a separate ticket.